build/parseSpec.cc

```diff
@@ -692,6 +692,9 @@ static int initSourceHeader(rpmSpec spec)
 
     /* Only specific tags are added to the source package header */
     headerCopyTags(spec->packages->header, sourcePkg->header, sourceTags);
+	char *nevr = headerGetAsString(sourcePkg->header, RPMTAG_NEVR);
+	headerPutString(spec->packages->header, RPMTAG_SOURCENEVR, nevr);
+	free(nevr);
```

Comment:
initSourceHeader() is the wrong place to do this, that function is about initializing the *source* header. finalizeSpec() would seem appropriate.

Also, the indentation is off here. See CONTRIBUTING.md on that.